packages/email/email.js

```diff
@@ -246,11 +249,18 @@ Email.send = function(options) {
     customTransport({ packageSettings, ...options });
     return;
   }
-  if (
-    Meteor.isProduction ||
-    process.env.MAIL_URL ||
-    Meteor.settings.packages?.email
-  ) {
+
+  const mailUrlEnv = process.env.MAIL_URL;
+  const mailUrlUrlSettings = Meteor.settings.packages?.email;
```

Comment:
Should't this variable be named `mailUrlSettings` instead of `mailUrlUrlSettings`?

Reply:
Yes, it was a typo